backends/core/models.py

```diff
@@ -32,6 +33,7 @@
 from core import inline
 from core.mailers import NotificationMailer
 
+MEMCACHE_DEFAULT_EXPIRATION_TIME = 24 * 60 * 60
```

Comment:
Maybe add a comment, or put `_SECONDS` in the name of the variable, to clarify the unit.

Reply:
done